src/writer.zig

```diff
@@ -137,7 +137,7 @@ fn send_pointer(buf: *ei.ei_x_buff, data: anytype) Error!void {
     }
 }
 
-pub fn send_payload(buf: *ei.ei_x_buff, data: anytype) Error!void {
+pub fn any(buf: *ei.ei_x_buff, data: anytype) Error!void {
```

Comment:
Why not `write_any`? I feel like `sender.send_payload` was giving the reader more information then `writer.any`. What do you think?